sgx-show/src/data/cpuid.rs

```diff
@@ -36,19 +36,19 @@ pub struct CpuId<T: Leaf>(pub u128, PhantomData<T>);
 
 impl<T: Leaf> CpuId<T> {
     pub fn eax(&self) -> u32 {
-        (self.0 >> 3 * 32) as u32
+        (self.0 >> (3 * 32)) as u32
     }
 
     pub fn ebx(&self) -> u32 {
-        (self.0 >> 2 * 32) as u32
+        (self.0 >> (2 * 32)) as u32
     }
 
     pub fn ecx(&self) -> u32 {
-        (self.0 >> 1 * 32) as u32
+        (self.0 >> 32) as u32
     }
 
     pub fn edx(&self) -> u32 {
-        (self.0 >> 0 * 32) as u32
+        self.0 as u32
```

Comment:
I think the code is more readable with the identity ops. And the compiler will optimize them out. So let's just disable the lint.